src/arithmetic/entity_funcs/entity_funcs.c

```diff
@@ -204,7 +204,15 @@ SIValue AR_PROPERTY(SIValue *argv, int argc, void *private_data) {
 
 	SIValue obj = argv[0];
 
-	if(SI_TYPE(obj) & SI_GRAPHENTITY) {
+	if(SI_TYPE(obj) == T_POINT)
+	{
+		// retrieve property key 
+		SIValue key = argv[1];
+		SIValue value;
+
+		return Point_GetCoordinate(obj, key);
+	}
+	else if(SI_TYPE(obj) & SI_GRAPHENTITY) {
```

Comment:
please keep the formatting
```
if() {
} else if() {
}
```

Reply:
Done in [666387d](https://github.com/RedisGraph/RedisGraph/pull/2524/commits/666387d61e00cc67bae2abd7f23a1fd8803902b6)